api-service/controller/oracle_database_agreements.go

```diff
@@ -90,6 +90,13 @@ func (ctrl *APIController) UpdateOracleDatabaseAgreement(w http.ResponseWriter,
 		return
 	}
 
+	if req.Unlimited {
```

Comment:
Why increase [cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity) inserting one `if` into each other while you can just put two conditions in and (`&&`)?

Reply:
if the ULA parameter is true also the basket parameter has to be true otherwise an error must be returned
->
    if ( Unlimited == true ) also ( CatchAll == true ) -> otherwise ERROR
->
    if ( Unlimited == true ) && ( CatchAll == false ) -> ERROR
->
    if ( Unlimited ) && ( ! CatchAll ) -> ERROR